specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7056,6 +7113,153 @@
+            "type": "string"
+          },
+          "description": "The classification(s) of the patch as provided by the patch publisher."
+        },
+        "rebootBehavior": {
+          "type": "string",
+          "description": "Describes the reboot requirements of the patch.",
+          "enum": [
+            "NeverReboots",
+            "AlwaysRequiresReboot",
+            "CanRequestReboot"
+          ],
+          "x-ms-enum": {
+            "name": "SoftwareUpdateRebootBehavior",
+            "modelAsString": true
+          }
+        },
+        "activityId": {
+          "type": "string",
+          "description": "The activity ID of the operation that produced this result. It is used to correlate across CRP and extension logs."
+        },
+        "publishedDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp when the repository published this patch."
+        },
+        "lastModifiedDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp of the last update to this patch record."
+        },
+        "assessmentState": {
+          "type": "string",
+          "description": "Describes the outcome of an install operation for a given patch.",
+          "enum": [
+            "Installed",
+            "Failed",
+            "Excluded",
+            "NotSelected",
+            "Pending",
+            "Available"
+          ],
+          "x-ms-enum": {
+            "name": "PatchAssessmentState",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Describes the properties of a Virtual Machine software patch."
+    },
+    "VirtualMachineAssessPatchesResult": {
+      "properties": {
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The overall success or failure status of the operation. It remains \"InProgress\" until the operation completes. At that point it will become \"Failed\", \"Succeeded\", or \"CompletedWithWarnings.\"",
+          "enum": [
+            "InProgress",
+            "Failed",
+            "Succeeded",
+            "CompletedWithWarnings"
+          ],
+          "x-ms-enum": {
+            "name": "PatchOperationStatus",
+            "modelAsString": true
+          }
+        },
+        "assessmentActivityId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The activity ID of the operation that produced this result. It is used to correlate across CRP and extension logs."
+        },
+        "rebootPending": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The overall reboot status of the VM. It will be true when partially installed patches require a reboot to complete installation but the reboot has not yet occurred.",
+          "enum": [
+            "true",
+            "false"
+          ],
```

Comment:
This is not really whats intended by the "avoid booleans" suggestion.  Typically if you can model something as "rebootStatus": "Pending"/"SomeOtherValue" then it should be modeled that way instead of a boolean.  Can you do that here?

Reply:
It is really just a boolean (did we detect that a reboot is needed?) so I'll revert it back to being a boolean type.